File: Data Structures & Algorithms/missing-number/submission-2.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = max(nums)
        if (int(n*(n + 1))/2) != sum(nums):
            return (int(n*(n + 1)/2)) - sum(nums)
        elif 0 not in nums:
            return 0
        else:
            return n + 1

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-2.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int maxi = 0;
        int mini = prices[0];

        for (int& sell : prices) {
            maxi = max(maxi, sell - mini);
            mini = min(mini, sell);
        }
        return maxi;
    }
};
